src/pharmpy/modeling/metabolite.py

```diff
@@ -12,22 +12,28 @@
 )
 
 from .error import set_proportional_error_model
-from .odes import _find_noncov_theta, add_individual_parameter
-from .parameters import set_initial_estimates
+from .odes import _find_noncov_theta, add_individual_parameter, set_first_order_absorption
+from .parameters import set_initial_estimates, set_lower_bounds, set_upper_bounds
 
 
-def add_metabolite(model: Model, drug_dvid: int = 1):
+def add_metabolite(model: Model, drug_dvid: int = 1, presystemic_fixed_drug: bool = False):
```

Comment:
Is presystemic fixed drug the correct term?

Reply:
Changed to "presystemic" as it's a bit simpler more in line with what was found in theory